### DISCUSSION TITLE
--cap-string error

### DISCUSSION THREAD

**Discussion by nuweq** [seleniumbase/SeleniumBase#2996] [Q&A] [answered]
Hi,

I received this error message when running my test on a remote Selenium grid:
![image](https://github.com/user-attachments/assets/ce2a93cb-73cb-49a4-a243-66223ab3ced3)
Here is my pytest command line:
pytest tests/regression --protocol=http --server=IP_ADDRESS --port=PORT --cap-string='{"browserName":"chrome","name":"test1"}' --browser=remote --maximize --reruns=1 --html=report.html

Can please help me with this.
Thanks.

seleniumbase version: 4.27.5
Python: 3.11

**Comment by mdmintz** [comment, marked as answer]
I didn't see any errors when running `pytest TEST.py --cap-string='{"browserName":"chrome","name":"test1"}'` (with the other Selenium Grid args). Make sure you copied correctly.
